src/main/java/alchemyplusplus/block/complex/distillery/DistilleryBlock.java

```diff
@@ -26,17 +26,14 @@ public DistilleryBlock(String blockname)
     public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer player, int meta, float b, float c, float g)
     {
         TileEntity tileEntity = world.getTileEntity(x, y, z);
-        if (tileEntity != null && !player.isSneaking())
```

Comment:
this is a basic bypass, normally used by `Pipes` & other `Blocks`, so you can for example attach an `Extra Utilities Retrieval Node` to this Block, since it's direction depends upon what side of the block you are targeting. otherwise we'd been to place the Node before we place the block, but in order to do that, we'd need to place something else there first, such as Dirt or Cobblestone, then place the Node, then break the Dirt/Cobblestone/etc. _THEN_ place the Distillery, and that's not even factoring in the maths for preventing a Player from placing a Block whenever they open the GUI, that's extra!